burstphoto/ContentView.swift

```diff
@@ -8,7 +8,7 @@ enum AppState {
 class AppSettings: ObservableObject {
```

Comment:
(Not related to this line, I am merely starting it here so that we have a thread to discuss in)

The new frequency approach does not handle slightly mismatched specular highlight areas well, and the failure mode is very ugly.

Crop of the two input frames to show the region and the misalignment:
![input 1](https://user-images.githubusercontent.com/48962821/219970923-86d8845e-a525-464b-8be6-b3674047abe4.jpg)
![input 2](https://user-images.githubusercontent.com/48962821/219970921-723bf5b2-51c2-4dca-961c-2e2051d6d3bf.jpg)

The result of the frequency merge with NR=21
![frequency merge nr=21](https://user-images.githubusercontent.com/48962821/219970982-ff166c99-b798-4ff5-92f8-c89cd123076b.jpg)

I've also ran it with much lower NR level (e.g. NR=6) and these artifacts are still present. Many have gotten less bad but are still very clearly visible.

Here is the same scene when merged with the spatial approach set to minimum robustness:
![spatial merge (min robustnes)](https://user-images.githubusercontent.com/48962821/219971017-4a803c31-da98-4690-80e8-d3e2738d6ac0.jpg)

These artifacts are quite bad and are present in other regions of the image. They are visible even when without zooming in the image.

Another similar artifact from elsewhere in the scene where the transition is not between clipped highlights and deep blacks, this is between non-clipped highlights and a middle grey (before the exposure is increased)
![artifact](https://user-images.githubusercontent.com/48962821/219971217-1fa7e591-61e5-4772-9c02-be56a3aaa607.png)


I haven't dived in yet to see if something can be changed in the code to prevent them, but if not then I think it's worth either:
1. Switching to spatial merging for there tiles,
2. Or, simply not merging anything on these tiles.

Reply:
This issue has been resolved in the commit [Improvement of specular highlights and performance](https://github.com/chris-rank/hdr-plus-swift/commit/6a5a10f8c3b55a07d0f8a7ac64ce4f744c9a57ab).